File: backend/DataAccess/DaoGenerated/システム共有/tm_afhanyo_mainDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             汎用メインマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afhanyo_mainDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afhanyo_main";
		public string hanyomaincd { get; set; }                   //汎用メインコード
		public int hanyosubcd { get; set; }                       //汎用サブコード
		public string hanyosubcdnm { get; set; }                  //汎用サブコード名称
		public string? kananm { get; set; }                       //カナ名称
		public string? shortnm { get; set; }                      //略称
		public int? keta { get; set; }                            //桁数
		public int? userryoikikaisicd { get; set; }               //ユーザ領域開始コード
		public bool iflg { get; set; }                            //INSERT可能フラグ
		public bool uflg { get; set; }                            //UPDATE可能フラグ
		public bool dflg { get; set; }                            //DELETE可能フラグ
		public string? biko { get; set; }                         //備考
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tm_afhanyo_mainDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             汎用メインマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tm_afhanyo_mainDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afhanyo_main"
        Public Property hanyomaincd As String                   '汎用メインコード
        Public Property hanyosubcd As Integer                       '汎用サブコード
        Public Property hanyosubcdnm As String                  '汎用サブコード名称
        Public Property kananm As String                       'カナ名称
        Public Property shortnm As String                      '略称
        Public Property keta As Integer                            '桁数
        Public Property userryoikikaisicd As Integer               'ユーザ領域開始コード
        Public Property iflg As Boolean                            'INSERT可能フラグ
        Public Property uflg As Boolean                            'UPDATE可能フラグ
        Public Property dflg As Boolean                            'DELETE可能フラグ
        Public Property biko As String                         '備考
    End Class
End Namespace
